include/eve/detail/shuffle_v2/shuffle_v2_fwd.hpp

```diff
@@ -50,7 +50,8 @@ namespace eve
 //!      0: noop - this shuffle does not require any assembly.
 //!      1: constant - this shuffle produces requires one constant value and no instructions.
 //!                    (at the moment this is only zeroing out a register)
-//!      2: one instruction - this shuffle requires one instruction, no masks.
+//!      2: one instruction - this shuffle requires one instruction, no vector masks.
+//!                           scalar registers and avx512 masks are allowed.
```

Comment:
If I don't allow avx512 masks then we would have to copy most of the masked/not masked instructions detection twice